libs/remix-lib/src/execution/txRunnerWeb3.ts

```diff
@@ -187,6 +204,89 @@ export class TxRunnerWeb3 {
         })
     })
   }
+
+  async sendUserOp (tx) {
+    console.log('sendUserOp------->')
+    console.log('tx------->', tx)
+    const localStorageKey = 'smartAccounts'
+    const PUBLIC_NODE_URL = "https://rpc.ankr.com/eth_sepolia"
+    const PIMLICO_API_KEY =''
+    const BUNDLER_URL = `https://api.pimlico.io/v2/sepolia/rpc?apikey=${PIMLICO_API_KEY}`
+    const network = 'sepolia'
+    const chain = chains[network]
+  
+    // @ts-ignore
+    const [account] = await window.ethereum!.request({ method: 'eth_requestAccounts' })
+    // Check that saOwner is there in MM addresses
+    let smartAccountsObj = localStorage.getItem(localStorageKey)
+    smartAccountsObj = JSON.parse(smartAccountsObj)
+    const saDetails = smartAccountsObj[chain.id][tx.from]
+    const saOwner = saDetails['ownerEOA']
+    
+    // both are needed. public client to get nonce and read blockchain. wallet client to sign the useroperation
+    const walletClient = createWalletClient({
+      account: saOwner,
+      chain,
+      transport: custom(window.ethereum!),
+    })
+
+    const publicClient = createPublicClient({ 
+      chain,
+      transport: http(PUBLIC_NODE_URL) // choose any provider here
+    })
+
+    const safeAccount = await toSafeSmartAccount({
+      client: publicClient,
+      entryPoint: {
+          address: entryPoint07Address,
+          version: "0.7",
+      },
+      owners: [walletClient],
+      version: "1.4.1",
+      address: tx.from // tx.from & saDetails['address'] should be same
+    })
+
+    const paymasterClient = createPimlicoClient({
+      transport: http(BUNDLER_URL),
+      entryPoint: {
+          address: entryPoint07Address,
+          version: "0.7",
+      },
+    })
+    const saClient = createSmartAccountClient({
+        account: safeAccount,
+        chain,
+        paymaster: paymasterClient,
+        bundlerTransport: http(BUNDLER_URL),
+        userOperation: {
+          estimateFeesPerGas: async () => (await paymasterClient.getUserOperationGasPrice()).fast,
+        }
+    })
+
+    let salt: `0x${string}` = "0x0000000000000000000000000000000000000000000000000000000000000013"
+    let bytecode = tx.data
+    
+    const expectedDeploymentAddress = getContractAddress({ 
+      bytecode: keccak256(bytecode), 
```

Comment:
check the doc, parameters seem wrong: https://viem.sh/docs/utilities/getContractAddress.html#opcode-optional